### DISCUSSION TITLE
[Weekly Review Report]: August 26 - 29

### DISCUSSION THREAD

**Discussion by fehrhart** [wikipathways/wikipathways-help#130] [curation-reports]
### Recent changes

 * Changes from curators, some new pathways - nothing special.

### Under construction

There are some pathways that look like students projects - can supervisors please check and approve?

Julia Bous - WP5432 Omega-3-fatty acids in senescence - finished?

Hafeez-ur-rahman - is WP5436 finished?

Anne Eggen 
- WP5454 Cirgarette Smoke Induced Lung Fibrosis
- WP5459 Isolated pulmonary embolism
- WP5468 Pro-inflammatory cytokines
- WP5477 Molecular pathway for oxidative stress 1.0
- WP5482 Oxidative stress in pulmonary tissue

Samuel Brusselmans 
- WP5457 Substance interaction with the pulmonary resident cell membrane components leading to pulmonary fibrosis - Carbon Nanotubes
- WP5462 PFAS interaction with liver PPARα receptor leading to hepatocellular adenomas and carcinomas (Homo sapiens)

Philip Ernst 
- WP 5460 AOP PM Asbestos and microfibre exposed pleural mesothelioma 
- WP5469 Hallmark of cancer: metastasis and epithelial-to-mesenchymal transition
- WP5475 Hallmark of cancer: sustaining proliferative signaling
- WP5483 Hallmark of cancer: non-mutational epigenetic reprogramming

Fien Van Genechten - WP5465 Fanconi anemia - we already have a FA pathway: WP3569 (Reactome)

### New pathways

[Homo sapiens:Inhibition of Gαq-regulated signaling in uveal melanoma](https://classic.wikipathways.org/index.php/Pathway:WP5490) ‎(27 August 2024 by [Khanspers](https://classic.wikipathways.org/index.php/User:Khanspers))
[Homo sapiens:Regulation of cytotoxic T cell responses by Malat1 - miR-15/16 circuit ](https://classic.wikipathways.org/index.php/Pathway:WP5489)‎(27 August 2024 by [Khanspers](https://classic.wikipathways.org/index.php/User:Khanspers))
[Homo sapiens:Cobalamin B12, Folate B Riboflavin B2 Pathway into Methionine Cycle & Folate Cycle](https://classic.wikipathways.org/index.php/Pathway:WP5488) ‎(26 August 2024 by [Alexmadsen1](https://classic.wikipathways.org/index.php/User:Alexmadsen1))

### Deleted Pathways

WP5436
WP5466 
for deletion
(I dont have deletion rights)

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @fehrhart! We discussed the incomplete student pathways in the weekly Community Curation meeting and believe several are students of @marvinm2.

As for [Homo sapiens:Inhibition of Gαq-regulated signaling in uveal melanoma](https://classic.wikipathways.org/index.php/Pathway:WP5490) ‎and [Homo sapiens:Regulation of cytotoxic T cell responses by Malat1 - miR-15/16 circuit ](https://classic.wikipathways.org/index.php/Pathway:WP5489)‎, I plan to complete those in the next week.
